int16 robot_id
State state
CellInfo[] lv_short
---
int16 next_horizon